test/e2e/azure_logcollector.go

```diff
@@ -356,6 +359,16 @@ func windowsNetworkLogs(execToPathFn func(outputFileName string, command string,
 	}
 }
 
+func windowsCrashDumpLogs(execToPathFn func(outputFileName string, command string, args ...string) func() error) []func() error {
+	return []func() error{
+		execToPathFn(
+			// capture any crashdump files created by windows into a .tar to be collected via sftp
+			"tar-crashdumps.txt",
+			"powershell", "$p = 'c:\\localdumps' ; if (Test-Path $p) { tar.exe -cvzf c:\\crashdump.tar $p *>&1 | %{ Write-Output \"$_\"} } else { Write-Host \"No crash dumps found at $p\" }",
```

Comment:
I think there's a typo here, `c:\\crashdump.tar` should be `c:\\crashdumps.tar` so that the E2E log collector will pick it up (or change the log collector filename above).

Reply:
Good catch and yes.
i was testing the sftp functionality with a different file that did exist on the machine :(
Also, I'm trying to figure out how to get log output from this task. If you have any ideas I'm all ears!